package.json

```diff
@@ -6,7 +6,7 @@
   "author": "The Gaia Team <dev-gaia@lists.mozilla.org>",
   "engineStrict": "true",
   "engines": {
-    "node": ">=0.10",
+    "node": ">=4.0",
```

Comment:
I think changing the node engine version to >=4.0 is fine. There's no reason to not support future versions if it keeps on compiling magically. When it breaks at a certain version we can cap which version we go up to.

Changing the gaia version can be done in this file as well, I would just bump it to 0.0.4 personally. I don't see any value in making it 4.0.0.